unpub/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM dart:stable AS build
+
+WORKDIR /app
+COPY pubspec.* ./
+# RUN dart pub get
```

Comment:
I am on Mac and Docker runs with a Linux kernel, `dart compile exe` generates a binary in host OS format and does not support cross platform compilation; we can move to the other approach but had to test if this works, if creating the image on Linux or a Linux instance from github actions, then it will be fine.